core/webui-clouddrives/src/main/java/org/exoplatform/services/cms/clouddrives/webui/watch/EmailNotifyCloudDocumentListener.java

```diff
@@ -110,6 +111,8 @@ public void onEvent(EventIterator arg0) {
     WatchCloudDocumentServiceImpl watchService =
                                                (WatchCloudDocumentServiceImpl) WCMCoreUtils.getService(WatchDocumentService.class);
     MessageConfig messageConfig = watchService.getMessageConfig();
+    String sender = MailUtils.getSenderName() + " <" + MailUtils.getSenderEmail() + ">";
```

Comment:
As it is done in NotificationPluginUtils.getFrom(), the space is not needed
```suggestion
    String sender = MailUtils.getSenderName() + "<" + MailUtils.getSenderEmail() + ">";
```

Reply:
ok